bitnami/phpmyadmin/templates/serviceaccount.yaml

```
@@ -0,0 +1,19 @@
+{{- if .Values.serviceAccount.create }}
+apiVersion: v1
+kind: ServiceAccount
+automountServiceAccountToken: {{ .Values.serviceAccount.automountServiceAccountToken }}
+metadata:
+  name: {{ template "phpmyadmin.serviceAccountName" . }}
+  namespace: {{ .Release.Namespace }}
```

I think in this line you need to quotes
```suggestion
  namespace: {{ .Release.Namespace | quote }}
```